src/containers/pages/FocusInvestigation/active/tests/index.test.tsx

```diff
@@ -434,4 +435,36 @@ describe('containers/pages/ActiveFocusInvestigation', () => {
         .prop('data')
     ).toEqual([selectedPlan1]);
   });
+
+  it('should handle falsy result correctly', async () => {
+    const supersetServiceMock: any = jest
+      .fn(() => Promise.resolve(fixtures.plans))
+      .mockImplementationOnce(async () => null);
```

Comment:
Why this two and `supersetService` is a function?
Can't we have `.fn(() => Promise.resolve(null))` only?

Reply:
The intention was to have the first call (Called in component did mount) to return null and the [default](https://jestjs.io/docs/en/mock-function-api#mockfnmockimplementationoncefn) implementation to return correctly. Your suggested implementation would still be ok